architecture/user_input/buttons.md

```diff
@@ -5,29 +5,24 @@
 Our first example is a simple counter that can be incremented or decremented. I find that it can be helpful to see the entire program in one place, so here it is! We will break it down afterwards.
 
 ```elm
-import Html exposing (Html, button, div, text)
-import Html.App as Html
+import Html exposing (div, button, text)
+import Html.App exposing (beginnerProgram)
 import Html.Events exposing (onClick)
 
 
 main =
-  Html.beginnerProgram { model = model, view = view, update = update }
+  beginnerProgram { model = 0, view = view, update = update }
 
 
 -- MODEL
 
-type alias Model = Int
-
-model : Model
-model =
-  0
```

Comment:
If I didn't know about types yet: `model = 0` and `Model = Int` I might think "two models!?"